client/extensions/woocommerce/state/wc-api/reducer.js

```diff
@@ -0,0 +1,40 @@
+/**
```

Comment:
I would have but I needed code from multiple Branch's so I would have had to rebase my changes on top of your branch.